Anytype/Sources/PresentationLayer/Modules/HomeWidgets/Widgets/List/Specific/List/SetListWidgetModuleAssembly.swift

```diff
@@ -27,7 +27,7 @@ final class SetListWidgetModuleAssembly: HomeWidgetCommonAssemblyProtocol {
             setSubscriptionDataBuilder: SetSubscriptionDataBuilder(
                 activeWorkspaceStorage: serviceLocator.activeWorkspaceStorage()
             ),
-            subscriptionService: serviceLocator.subscriptionService(),
+            subscriptionStorageProvider: serviceLocator.subscriptionStorageProvider(),
```

Comment:
`reminder`: use subscription service in my current PR with templates - needs to be updated too (by me or you)